src/veh_interact.cpp

```diff
@@ -734,6 +737,17 @@ void veh_interact::do_install()
 
     int pos = 0;
     size_t tab = 0;
+
+    std::string sFilter;
+    std::function<bool(const vpart_info*)> user_filter = [&](const vpart_info *p) { auto &part = *p;
+        return lcmatch(part.name, sFilter);
```

Comment:
Just `lcmatch( p->name, ...` is good, no need to introduce the temporary `part` reference.